casa/System/Aipsrc.cc

```diff
@@ -293,59 +294,61 @@ uInt Aipsrc::registerRC(const String &keyword,
 			Int Nname, const String tname[], 
 			const String &deflt) {
   uInt n = Aipsrc::registerRC(keyword, ncodlst);
-  codlst.resize(n);
+  if(n > codlst.size())
+    codlst.resize(n);
```

Comment:
Since `registerRC` already resizes the list, can't you simply remove this code?
```suggestion
```

Reply:
Als here, registerRC resizes `ncodlst`, not `codlist`.